### ISSUE TITLE
Not able to scan the dependencies

### ISSUE THREAD

**Issue by Mridulau** [dependency-check/DependencyCheck#5631]
Hi @jeremylong 

We are using Dependency-check version 6.5.3 to scan the code, but in the output we don't see any results in the report.
Can you let us know what could be the reason for it.
I have attached the output logs of dependency check and html report. Please verify

Thank you,
Mridula U
[ESB_Dep_logs.txt](https://github.com/jeremylong/DependencyCheck/files/11136143/ESB_Dep_logs.txt)

**Comment by jeremylong** [comment]
6.5.3 is not a supported version of dependency-check.  Also - without knowing anything about the project being scanned we can't really even guess as to why you get no results.

**Comment by Mridulau** [comment]
@jeremylong 

Please let me know what information is required from my end

We are trying to scan the Java project using the below command:
<img width="587" alt="Java_project" src="https://user-images.githubusercontent.com/43175836/229505806-a5706c3f-4a49-46de-8ebc-e5ebd85f4c5d.png">

/apps/installers/dependency-check-6.5.3-release/bin/dependency-check.sh --noupdate --disableRetireJS --disableCentral --disableOssIndex --disableYarnAudit --scan . --project Batch-ESB --proxyserver 172.25.1.23 --proxyport 8080 --format ALL --out /workspace/BatchArch/ESB/ESB-Fortify-Report/devopsDir/depcheck

**Comment by Mridulau** [comment]
Hi @jeremylong 

Any updates on the request??

**Comment by jeremylong** [comment]
I have no clue what the project is - maven, gradle, both? Possibly add a task to the grale project (if you can't modify the project - you could use an init script for this):

```
task copyDependencies(type: Copy) {
  from configurations.default
  into 'dependencies'
}
```

Then run:

```bash
$ mvn dependency:copy-dependencies
$ ./gradlew copyDependencies
```

And then scan with dependency check. A better bet would be to talk to the developers, add the maven or gradle plugin as appropriate to the project.... Or just use the ODC gradle plugin (via an init script) and the maven plugin by specifying it on the command line.

**Comment by Mridulau** [comment]
Hi @jeremylong 

We have converted the project to gradle project to avoid confusion. Now we are able to see the dependencies in HTML report where it is flagging gradle-wrapper.jar as dependency which is not correct

We have defined the dependencies in build.gradle file but why it not flagging these below dependencies in the report?

I have attached the content of build file and the way we have configured dependency check to scan. 
[dependencycheck.zip](https://github.com/jeremylong/DependencyCheck/files/11219080/dependencycheck.zip)
